icon-themes/ContrastHigh/16x16/devices/Makefile.am

```diff
@@ -53,6 +53,8 @@ icon_DATA=	\
 	printer-network.png \
 	printer.png \
 	scanner.png \
-	video-display.png
+	video-display.png \
+        nm-device-wired.png \
+        nm-device-wireless.png
```

Comment:
IMO you should keep the items sorted, e.g. add those after *nm-applet.png* above, and same in the other files.